Service.Core/Kernel/Kernel.Core/ArgParser.hpp

```diff
@@ -32,16 +32,16 @@ class ArgParser
     */
     explicit ArgParser(int argc, const char** argv) 
     {
-        _argParser = argparse::ArgumentParser("ServerArguments", "0.6",
+        _argParser = argparse::ArgumentParser("KernelArguments", "0.6",
         argparse::default_arguments::none);
 
         _argParser.set_assign_chars("=");
-        _argParser.add_description("Server side of juniorgram messenger. Just have a fun!");
+        _argParser.add_description("Kernel side of juniorgram messenger. Just have a fun!");
```

Comment:
"Backend service side of Juniorgram application. Just have a fun!"

Reply:
Done